components/onboarding/onboardingContext.js

```diff
@@ -70,8 +80,8 @@ function useOnboardingProvider() {
         // start the tour if the instance is created, which is the case after a route change
         if (tourInstance) {
             tourInstance.drive();
-            setCurrentStepDescription(tourInstance.getActiveStep().popover.description);
-            setCurrentStepTitle(tourInstance.getActiveStep().popover.title);
+            setCurrentStepDescription(tourInstance?.getActiveStep()?.popover?.description);
+            setCurrentStepTitle(tourInstance?.getActiveStep()?.popover?.title);
```

Comment:
tourInstance doesn't need a `?`, you're already  in a conditional `if(tourInstance) {...}`